public/src/listeners.js

```diff
@@ -43,23 +43,6 @@ export default function startListeners() {
         fullView();
     });
 
-    $('#projectName').on('click',() => {
-        simulationArea.lastSelected = globalScope.root;
-        setTimeout(() => {
-            document.getElementById("projname").select();
-        }, 100);
-    });
-    /* Makes tabs reordering possible by making them sortable */
-    $("#tabsBar").sortable({
```

Comment:
@dipanshparmar why is this code removed?

Reply:
@satu0king this code was used to handle the click on the header project title.
![image](https://user-images.githubusercontent.com/49204837/102004138-09fb7400-3d34-11eb-95e3-e27e7abfb16a.png)
Like when someone clicks on the project title it activates the project input section in the right to enter the project name.
But as this is no longer required because there is title input in the header itself now that's why this code is removed.
***
And I guess I accidentally removed the block of $('#tabsBar").sortable(). I'll restore this one.